packages/core/src/application.ts

```diff
@@ -26,6 +24,28 @@ export class Application extends Context {
         this.component(component);
       }
     }
+
+    if (options.servers) {
+      for (const server of options.servers) {
+        // server is a constructor
+        if (typeof server === 'function') {
+          this.server(server as Constructor<Server>);
+        } else {
+          // server is a name and ctor pair
+          const conf = server as NamedServer;
+          if (conf.name && conf.ctor) {
+            this.server(conf.ctor, conf.name);
+          } else {
+            const namedFormat = {
+              name: 'instanceName',
+              ctor: Function,
+            };
+            throw new Error(`servers array must contain Server constructors
+            or instances of the format: ${JSON.stringify(namedFormat)}`);
```

Comment:
I believe backticks preserve spacing in multi-line strings. As a result, this error message will contain about 14 space characters after `constructors`.

Reply:
Yikes. :S